mixer/pkg/runtime/requestDataBuilder.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2017 Istio Authors
```

Comment:
i'd consider calling this file simply `context.go`. It is shorter, easier to identify, and allows any other runtime context manipulation funcs to live here.